### PR TITLE
Block jump integration callback

### PR DESCRIPTION
**Pull request by 44jovi**
`feat(CookieJar; BlockJumpGame): integrate BlockJumpGame with main game via CookieJar`

### CODE REVIEW

**Comment by 44jovi** [comment]
[Integrate: BlockJumpGame to main game](https://trello.com/c/Eu5Y5eCQ/20-integrate-blockjumpgame-to-main-game)

File: index.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ const Level = require("./lib/level");
 const levelPlans = require("./lib/levelPlans");
 const DOMDisplay = require("./lib/DOMDisplay");
 const Game = require("./lib/game");
+const BlockJumpGame = require("./lib/blockJumpGame");
+const DumbMiniGame = require("./lib/dumbMiniGame");
```
Comment (ritaaktay) [inline index.js:6]: No need to require these anymore!

File: lib/blockJumpGame.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    this.character = document.getElementById("character");
+    this.block = document.getElementById("block");
+    this.jumpButton = document.getElementById("jump-button");
+    this.startButton = document.getElementById("start-button");
+    this.startButton.addEventListener("click", this.start);
+    this.jumpButton.addEventListener("click", this.jump);
+    this.jumpCounter = 0;
+    this.callback;
+    // When new instance of game is created, calls checkIfDead immediately
+    this.checkIfDead();
```
Comment (ritaaktay) [inline lib/blockJumpGame.js:12]: I would move this inside the run() method

File: lib/cookieJar.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+const Vec = require("./vector");
+const BlockJumpGame = require("./blockJumpGame");
+const State = require("./state");
+const DumbMiniGame = require("./dumbMiniGame");
```
Comment (ritaaktay) [inline lib/cookieJar.js:4]: We don't need to require this!

**Pull request merged**